babel.config.js

```diff
@@ -15,4 +15,12 @@ module.exports = {
             },
         ],
     ],
+    plugins: [
+        [
+            require.resolve('babel-plugin-styled-components'),
```

Comment:
what is the purpose of this change?

It seems like display name is true by default in this plugin.

It is a good opportunity to fix our tests for each PR - please make a separate PR and make adjustment for Jest with https://styled-components.com/docs/tooling#usage - Server-side rendering